types/jsdom/jsdom-tests.ts

```diff
@@ -1,9 +1,12 @@
 import "./test/core";
 import jsdom = require("jsdom");
+// tslint:disable-next-line:multiple-import-in-one-file test named export
+import { JSDOM } from "jsdom";
 
-const dom = new jsdom.JSDOM();
+const dom = new jsdom.JSDOM() || new JSDOM();
 const domWindow = dom.window; // $ExpectType DOMWindow
-
+dom.virtualConsole; // $ExpectType VirtualConsole
+dom.cookieJar; // $ExpectType CookieJar
 domWindow.document.querySelector("slot"); // $ExpectType HTMLSlotElement | null
 domWindow.AbstractRange.prototype; // $ExpectType AbstractRange
 domWindow.StaticRange.prototype; // $ExpectType StaticRange
```

Comment:
This should probably be done in a separate PR.

Reply:
this was mix of scout badge and avoiding untested badge fate